webpack.config.js

```diff
@@ -30,11 +34,55 @@ module.exports = {
       }]
     },
     {
-      test: /\.(png|bin)$/,
+      test: /\.png$/,
       loader: "url-loader",
       options: {
-        limit: true, // everything
+        limit: 8192,
+        outputPath: 'assets/images',
+        publicPath: 'images',
+        postTransformPublicPath: (p) => `__ml_activities_asset_public_path__ + ${p}`,
+        name: '[name].[ext]?[contenthash]',
```

Comment:
The use of `[contenthash]` here ensures that the request URLs encoded into the JS code will change in the event that the content of the files change. This is very helpful in case we modify assets without changing filenames and re-deploy.